src/wstool/cli_common.py

```diff
@@ -162,12 +162,18 @@ def _get_status_flags(basepath, elt_dict):
          elt_dict['actualversion'] is not None and
          elt_dict['specversion'] != elt_dict['actualversion'])):
         mflag += 'V'
-    if ('remote_revision' in elt_dict and
-        elt_dict['remote_revision'] != '' and
-        elt_dict['remote_revision'] is not None and
-        elt_dict['actualversion'] is not None and
-        'actualversion' in elt_dict and
-        elt_dict['remote_revision'] != elt_dict['actualversion']):
+    if (('remote_revision' in elt_dict and
+         elt_dict['remote_revision'] != '' and
+         elt_dict['remote_revision'] is not None and
+         elt_dict['actualversion'] is not None and
+         'actualversion' in elt_dict and
```

Comment:
Well, if you like you can swap the order of the checks or remove the existence check, but I'm fine with leaving it as it is as well.